LMG/FAQ.md

```diff
@@ -27,21 +27,73 @@ ms.search.validFrom: 2016-05-31
 ms.dyn365.ops.version: AX 7.0.1
 ---
 
-# 	Frequently Asked Questions
+# Frequently Asked Questions
+
+<!-- TOC -->
+
+FAQs
+
+- [Version and Support Information](#version-and-support-information)
+- [Dependency and Security Information](#dependency-and-security-information)
+- [Troubleshooting](#troubleshooting)
+
+<!-- /TOC -->
 
 ## Version and Support Information
+
 Where can I find contact information for support?
 Email: ECLANZProductSupport@dxc.com
 
-## 	Troubleshooting
-1. Prompt with fix connection when accessing the Power App.   
-![License Manager PowerApp Connection Failure](IMAGES/PowerAppConnectionFailure.png "PowerAppConnectionFailure")
-The Power App platform requires consent to authenticate and provide access to the license manager service. This prompt appears everytime a new session is started to an existing session expires due to no user input. The standard expiration time for the access token is 60 minutes. While the app is in use, it will be able to auto-refresh the access token.  
+## Dependency and Security Information
+
+- The License Manager customization for ISV product license validation is implemented within the existing Dynamics 365 Finance and Operations framework.
+- Does not include reference to any other third party assembly files.
```

Comment:
Can't write that, we have to state that we use azure application insight lib

Reply:
Modified the content, ready for review.